install_clam.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+apt-get update -qq
+apt-get install -qq clamav-daemon
+echo TCPSocket 3310 >> /etc/clamav/clamd.conf
```

Comment:
Personally, I'd prefer if these were kept as distinct steps like before in `.travis.yml` as it makes slightly easier reading. Also, the echoing of the socket into the config file could be done as `echo "TCPSocket 3310" | sudo tee -a /etc/clamav/clamd.conf` - I know that appending with sudo is a bit finicky to work out sometimes!